packages/cardpay-sdk/sdk/utils/signing-utils.ts

```diff
@@ -72,7 +72,8 @@ export async function signSafeTx(
   estimate: Estimate,
   nonce: BN,
   from: string,
-  signer?: Signer
+  signer?: Signer,
+  refundReceiver?: string
```

Comment:
Doesn’t the `estimate` object already include the `refundReceiver`? Is there a reason you make all the callers pass this is separately? Could we not just pull out the `refundReceiver` from the `estimate`?